Gemfile

```diff
@@ -16,6 +16,11 @@ end
 ### dep for ci/coverage
 gem 'simplecov', '~> 0.8'
 
-gem 'rubocop', "~> 0.23.0", :platform => [:ruby_19, :ruby_20, :ruby_21]
+# There is no platform :ruby_193 and Rubocop only supports >= 1.9.3
+unless RUBY_VERSION == "1.9.2"
+  gem "rubocop",
+      "~> 0.32.1",
+      :platform => [:ruby_19, :ruby_20, :ruby_21, :ruby_22]
```

Comment:
Unfortunately no.

```
[!] There was an error parsing `Gemfile`: You cannot specify the same gem twice with different version requirements.
You specified: rubocop (~> 0.23.0) and rubocop (~> 0.32.1). Bundler cannot continue.

 #  from /rspec-dev/repos/rspec-support/Gemfile:26
 #  -------------------------------------------
 #  gem "rubocop", "~> 0.23.0", :platform => [:ruby_19]
 >  gem "rubocop", "~> 0.32.1", :platform => [:ruby_20, :ruby_21, :ruby_22]
 #  
 #  -------------------------------------------
```

I did a brief test and some of the cops / settings are not backwards compatible. I'm not sure I see a good way to keep this running on 1.9.2 with an older Rubocop.